sockeye/train.py

```diff
@@ -723,6 +724,13 @@ def main():
     arguments.add_train_cli_args(params)
     args = params.parse_args()
 
+    if args.dry_run:
+        # Modify arguments so that we write to a temporary directory and
+        # perform 0 training iterations
+        temp_dir = tempfile.TemporaryDirectory()  # Will be automatically removed
```

Comment:
Are you sure that this gets removed?

Reply:
From the documentation: "On completion of the context or destruction of the temporary directory object the newly created temporary directory and all its contents are removed from the filesystem." (https://docs.python.org/3/library/tempfile.html)

I also checked after completion and it was indeed removed.

Reply:
Ok, thanks!